components/stacklayout/overview.md

```diff
@@ -10,24 +10,19 @@ position: 0
 
 # Blazor StackLayout Overview
 
-The <a href="https://www.telerik.com/blazor-ui/stacklayout" target="_blank">Blazor StackLayout component</a> allows you to easily align multiple elements in a vertical or horizontal order. 
+The <a href="https://www.telerik.com/blazor-ui/stacklayout" target="_blank">StackLayout for Blazor</a> is a component that easily allows aligning multiple elements in a vertical or horizontal order.
 
-#### In This Article:
+## Creating Blazor StackLayout
 
+1. Use the `<TelerikStackLayout>` tag to add the component to your razor page.
 
-* [Basics](#basics)
-* [Features](#features)
-* [Nested StackLayouts](#nested-stacklayouts)
+2. Inside the `<TelerikStackLayout>` tag, add the desired custom elements.
```

Comment:
This step is not actionable. What are these custom elements? Or what can they be?